pages/blog/release-notes-2.2.0.md

```diff
@@ -60,4 +60,4 @@ Are you wondering how we managed to release 2.2.0 just three months after 2.1.0?
 
 Does the above meme give you mixed feelings? Are you afraid of possible changes, or actually happy to see it coming? Don't overthink it! Join our [Slack](https://www.asyncapi.com/slack-invite) and talk to us, or check out the [3.0.0 milestone](https://github.com/asyncapi/spec/milestone/18).
 
-> Photo by <a href="https://unsplash.com/@jeremythomasphoto?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Jeremy Thomas</a> on <a href="https://unsplash.com/s/photos/autumn?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
```

Comment:
I really don't know what's the change here. Maybe there is a single character difference in the file but I'm not getting the exact diff here. Hope that you won't consider it much :slightly_smiling_face:. Same goes for rest specification file and release notes file.